taxi/views.py

```diff
@@ -38,18 +40,18 @@
 class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
     model = Manufacturer
     fields = "__all__"
-    success_url = reverse_lazy("taxi:manufacturer-list")
+    success_url = reverse_lazy("taxi:manufacturer_list")
 
 
 class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
     model = Manufacturer
     fields = "__all__"
-    success_url = reverse_lazy("taxi:manufacturer-list")
+    success_url = reverse_lazy("taxi:manufacturer_list")
 
 
 class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
     model = Manufacturer
-    success_url = reverse_lazy("taxi:manufacturer-list")
+    success_url = reverse_lazy("taxi:manufacturer_list")
```

Comment:
The URL name in `reverse_lazy` function should be written with `-` instead of `_`. Please, change `manufacturer_list` to `manufacturer-list`.